src/Scripts/WindowsLocalTest.ps1

```diff
@@ -0,0 +1,132 @@
+    The COR_PROFILER_PATH string for the raw profiler dll.
+    NOTE: This config file will be set for both the 32-bit and 64-bit environment variable.
+
+.PARAMETER DebugWait
+    Optional, sets MicrosoftInstrumentationEngine_DebugWait which causes the ClrInstrumentationEngine to wait for a debugger to attach.
+
+.PARAMETER ProxyUseDebug
+    Optional, forces the CLRIE ProfilerProxy to look at CLRIE versions with "_Debug" suffix.
+
+.PARAMETER ProxyUsePreview
+    Optional, allows the CLRIE ProfilerProxy to look at both release and preview (with "-build#####" suffix) versions.
+#>
+
+[CmdletBinding(DefaultParameterSetName='None')]
+param(
+    [Parameter(Mandatory = $false)]
+    [ValidateNotNullOrEmpty()]
+    [ValidateScript({ Test-Path $_ })]
+    [string]
+    $ApplicationPath = 'powershell',
+
+    [Parameter(ParameterSetName='InstrumentationMethod', Mandatory = $true)]
+    [Parameter(ParameterSetName='RawProfilerHook', Mandatory = $false)]
+    [ValidateNotNullOrEmpty()]
+    [ValidateScript({ Test-Path $_ })]
+    [string]
+    $InstrumentationMethodConfigPath,
+
+    [Parameter(ParameterSetName='InstrumentationMethod', Mandatory = $false)]
+    [Parameter(ParameterSetName='RawProfilerHook', Mandatory = $true)]
+    [ValidateNotNullOrEmpty()]
+    [string]
+    $RawProfilerHookGuid,
+
+    [Parameter(ParameterSetName='InstrumentationMethod', Mandatory = $false)]
+    [Parameter(ParameterSetName='RawProfilerHook', Mandatory = $true)]
+    [ValidateNotNullOrEmpty()]
+    [ValidateScript({ Test-Path $_ })]
+    [string]
+    $RawProfilerHookPath,
+
+    [Parameter(Mandatory = $false)]
+    [switch]
+    $DebugWait,
+
+    [Parameter(Mandatory = $false)]
+    [switch]
+    $DisableSignatureValidation,
+
+    [Parameter(Mandatory = $false)]
+    [switch]
+    $ProxyUseDebug,
+
+    [Parameter(Mandatory = $false)]
+    [switch]
+    $ProxyUsePreview
+)
+
+# COR_PROFILER variables
+$env:COR_ENABLE_PROFILING = 1
+$env:COR_PROFILER = "{324F817A-7420-4E6D-B3C1-143FBED6D855}"
+$env:COR_PROFILER_PATH_32 = "C:\Program Files (x86)\Microsoft CLR Instrumentation Engine\Proxy\v1\InstrumentationEngine.ProfilerProxy_x86.dll"
+$env:COR_PROFILER_PATH_64 = "C:\Program Files\Microsoft CLR Instrumentation Engine\Proxy\v1\InstrumentationEngine.ProfilerProxy_x64.dll"
+$env:CORECLR_ENABLE_PROFILING = 1
+$env:CORECLR_PROFILER = "{324F817A-7420-4E6D-B3C1-143FBED6D855}"
+$env:CORECLR_PROFILER_PATH_32 = "C:\Program Files (x86)\Microsoft CLR Instrumentation Engine\Proxy\v1\InstrumentationEngine.ProfilerProxy_x86.dll"
+$env:CORECLR_PROFILER_PATH_64 = "C:\Program Files\Microsoft CLR Instrumentation Engine\Proxy\v1\InstrumentationEngine.ProfilerProxy_x64.dll"
+
+# CLRIE variables
+if ($DebugWait) {
+    $env:MicrosoftInstrumentationEngine_DebugWait = 1
+} else {
+    $env:MicrosoftInstrumentationEngine_DebugWait = 0
+}
+
+if ($DisableSignatureValidation) {
+    $env:MicrosoftInstrumentationEngine_DisableCodeSignatureValidation = 1
```

Comment:
IIRC we don't check the actual value, just the existence of the environment variable.

Reply:
You are correct

---
In reply to: [330623962](https://github.com/microsoft/CLRInstrumentationEngine/pull/139#discussion_r330623962) [](ancestors = 330623962)